packages/legacy/core/App/navigators/TabStack.tsx

```diff
@@ -53,11 +63,11 @@ const TabStack: React.FC = () => {
               <AttachTourStep tourID={TourID.HomeTour} index={1}>
                 <View style={{ ...TabTheme.tabBarContainerStyle, justifyContent: showLabels ? 'flex-end' : 'center' }}>
                   <Icon name={focused ? 'message-text' : 'message-text-outline'} color={color} size={30} />
+
                   {showLabels && (
                     <Text
                       style={{
                         ...TabTheme.tabBarTextStyle,
-                        color: focused ? TabTheme.tabBarActiveTintColor : TabTheme.tabBarInactiveTintColor,
```

Comment:
Just curious, why did we get rid of this line?

Reply:
I see it locally. Looking to see why.

Reply:
Fixed.